projects/admin/src/app/record/formly/type/entity-typeahead/entity-typeahead.component.ts

```diff
@@ -72,6 +75,19 @@ export class EntityTypeaheadComponent extends FieldType implements OnInit {
        return this._remoteTypeahead
          .getSuggestions(options, query, this._numberOfSuggestions)
          .pipe(
+           map((suggestions: any) => {
+            // If we have no results or no local suggestion
+            // We create a local suggestion to display the section with the button.
```

Comment:
we create a local suggestion ... --> we create a fake local suggestion... this suggestion will not be rendered but is necessary to build the section header.